packages/odyssey-react-mui/src/Tabs.tsx

```diff
@@ -171,12 +183,20 @@ const Tabs = ({
     [tabState],
   );
 
+  let scrollButtons: MuiTabListProps["scrollButtons"] = "auto";
+  if (scrollButtonsDisplay === "always") {
+    scrollButtons = true;
+  } else if (scrollButtonsDisplay === "never") {
+    scrollButtons = false;
+  }
```

Comment:
We shouldn't need to use `let` here. I think a ternary should suffice, but I see that we have `else if`. Instead, we could use an object; and that'd be better for this situation:
```
const muiScrollButtonsProp: Record<scrollButtonsDisplay, boolean | undefined> = {
  always: true,
  auto: undefined,
  never: false,
}
```

Then, do get the value, all you need to do is:
```
scrollButtons={muiScrollButtonsProp[scrollButtonsDisplay]}
```